src/Sarif.Driver/Sdk/AnalyzeCommandBase.cs

```diff
@@ -305,6 +305,11 @@ private ISet<string> ValidateTargetsExist(TContext context, ISet<string> targets
                 Policy = policy ?? new PropertiesDictionary()
             };
 
+            context.Traces =
```

Comment:
>context [](http://example.com/codeflow?start=12&length=7)

This handling can now be removed from upstream driver tools that are currently required to do the same thing.